PULL_REQUEST_TEMPLATE

```diff
@@ -1,24 +1,32 @@
 Jira: [JIRA-TOKEN](https://ticketevolution.atlassian.net/browse/JIRA-TOKEN)
 
+> [!IMPORTANT]
+> Toggle to SQUASH AND MERGE when merging PRs to `staging` branch
+
 ## Why / What
+Why: These changes are for...
+
+What: This PR adds/removes/fixes... 
 
-This PR adds/removes/fixes&hellip;
+If this PR requires other developers to make changes to their local environment after this PR has been merged, please
+describe here. Examples: DB migration, Docker container restart/rebuild, .env changes etc.
 
 ## Testing
 
-I tested this code by&hellip;
+I tested this code by (unit/integration/manual) testing...
+
+## Environment Variables
+
+If there are any changes to environment variables, list here and link any relevant `tevo-kubernetes` PRs. Update the 
+project's .env file in 1Password after PR has been merged if necessary.
```

Comment:
This stuff is also on the JIra ticket, or should be. why should we duplicate here?

Reply:
I think the
> list here and link any relevant `tevo-kubernetes` PRs. Update the project's .env file in 1Password after PR has been merged if necessary.

part is important for more immediate visibility and as a reminder

Reply:
I do think the reminder/relationship is useful Nathan, but that's Jira's job and these're just words I'm going to begrudge deleting 8 times a week. IMO the template should be headers only (few agree afaik), but we definitely shouldn't be adding content with the intent that the PR author remove these characters.